### PR TITLE
Configuration in test is not based on properties file

### PR DESCRIPTION
**Pull request by kunsingh**
This change is [Reviewable](https://reviewable.io/reviews/canoo/dolphin-platform/824)

### CODE REVIEW

File: platform/dolphin-platform-server/src/test/java/com/canoo/impl/server/ClasspathScannerTest.java [hunk 1]
```diff
@@ -36,7 +39,8 @@
     @Test
     public void testSimpleScan() {
         //There can't be a class that is annotated with Inject
-        final DefaultClasspathScanner scanner = new DefaultClasspathScanner();
+        DefaultPlatformConfiguration defaultPlatformConfiguration = ConfigurationFileLoader.loadConfiguration();
```
Comment (hendrikebbers) [inline platform/dolphin-platform-server/src/test/java/com/canoo/impl/server/ClasspathScannerTest.java]: final

File: platform/dolphin-platform-spring-controller-tester/src/main/java/com/canoo/impl/dp/spring/test/TestConfiguration.java [hunk 2]
```diff
@@ -53,6 +55,9 @@ public TestConfiguration(final WebApplicationContext context, final HttpSession
         Assert.requireNonNull(context, "context");
         Assert.requireNonNull(httpSession, "httpSession");
 
+        DefaultPlatformConfiguration defaultPlatformConfiguration = ConfigurationFileLoader.loadConfiguration();
+        RemotingConfiguration remotingConfiguration = new RemotingConfiguration(defaultPlatformConfiguration);
```
Comment (hendrikebbers) [inline platform/dolphin-platform-spring-controller-tester/src/main/java/com/canoo/impl/dp/spring/test/TestConfiguration.java]: final

File: platform/dolphin-platform-spring-controller-tester/src/main/java/com/canoo/impl/dp/spring/test/TestConfiguration.java [hunk 3]
```diff
@@ -61,13 +66,13 @@ public TestConfiguration(final WebApplicationContext context, final HttpSession
...
         //Server
-        final ControllerRepository controllerRepository = new ControllerRepository(new DefaultClasspathScanner());
+        final ControllerRepository controllerRepository = new ControllerRepository(new DefaultClasspathScanner(defaultPlatformConfiguration.getListProperty(ROOT_PACKAGE_FOR_CLASSPATH_SCAN)));
         final TestSpringManagedBeanFactory containerManager = new TestSpringManagedBeanFactory(context);
         containerManager.init(context.getServletContext());
         final DolphinContextProviderMock dolphinContextProviderMock = new DolphinContextProviderMock();
 
 
-        dolphinTestContext = new TestDolphinContext(new RemotingConfiguration(ConfigurationFileLoader.loadConfiguration()), new HttpClientSessionImpl(httpSession), dolphinContextProviderMock, containerManager, controllerRepository, createEmptyCallback());
+        dolphinTestContext = new TestDolphinContext(remotingConfiguration, new HttpClientSessionImpl(httpSession), dolphinContextProviderMock, containerManager, controllerRepository, createEmptyCallback());
```
Comment (hendrikebbers) [inline platform/dolphin-platform-spring-controller-tester/src/main/java/com/canoo/impl/dp/spring/test/TestConfiguration.java]: final

**Comment by hendrikebbers** [comment]
@kunsingh can you please link this to an issue

**Comment by kunsingh** [comment]
https://github.com/canoo/dolphin-platform/pull/824

**Comment by hendrikebbers** [comment]
@kunsingh one build is still broken. Will just restart it

**Pull request merged**